### PR TITLE
chore(release): pull main into develop post release v1.81.0

### PR DESCRIPTION
**Pull request by devops-github-rudderstack**
:crown: *An automated PR*

### CODE REVIEW

**Comment by devops-github-rudderstack** [comment]
Test report for this run is available at: https://test-integrations-dev.s3.amazonaws.com/integrations-test-reports/rudder-transformer/3783/test-report.html

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,25 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [1.81.0](https://github.com/rudderlabs/rudder-transformer/compare/v1.79.1...v1.81.0) (2024-10-04)
+
+
```
Comment (krishna2020) [inline CHANGELOG.md:7]: @utsabc duplicate changelog of 1.80.0, need to be cleaned

Comment (utsabc) [reply CHANGELOG.md:7]: Is the process 
- After merging to develop cleaning it?
- Or use some admin privileges to clean it directly in main?

Comment (krishna2020) [reply CHANGELOG.md:7]: Ideally in the hotfix release branch, it has to be cleaned up.
Now, we can do it after merging to develop & later on it gets propagated to main as part of next release

**Review by krishna2020 — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (utsabc) [reply CHANGELOG.md:7]: ok

**Review by utsabc — APPROVED** [review]

**Pull request merged**